translations/strings.yaml

```diff
@@ -63,3 +63,22 @@ settingsDev:
   useStagingTitle: Use Staging Servers
   useStagingSubtitle: Staging services are used to test features before they go live to production
   restartRequired: VPN must be restarted to take effect
+
+splittunnel:
+  missingAppAction:
+    value:
+      - Add apps manually
+    comment:
+      - Refers to the Apps that are Missing from the Split Tunnel list
+  missingAppOne: 
+    value:
+      - \%0 Apps missing from list
+    comment:
+      - Refers to the Apps that are Missing from the Split Tunnel list
+  missingAppMultiple: 
+    value:
+      - \%0 Apps missing from list.
```

Comment:
Why this % is escaped and not the others?

Reply:
a value beginning with % is reserved for "%TAG" and "%YAML" directive, so the parser throws if i dont escape it